bpython/repl.py

```diff
@@ -1009,6 +1010,10 @@ def undo(self, n=1):
 
         entries = list(self.rl_history.entries)
 
+        #Most recently undone command
+        lastEntry = self.history[-n:]
```

Comment:
Could you change lastEntry -> last_entries; plural because it's a list, and snake_case